client/src/components/AddDataForm/index.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Form, Input, Button, message } from 'antd';
+import './style.css';
+
+const axios = require('axios');
+
+const AddDataForm = ({ formType, apiLink, cohortId }) => {
+  const onFinish = async (values) => {
+    try {
+      const sendValues = values;
+      sendValues.cohortId = cohortId;
```

Comment:
We will not send cohort id  for each post request, so you need to add condition depending on form type before this step

Reply:
yeah   😁